manifests/deps/debian.pp

```diff
@@ -3,6 +3,11 @@
 # This module manages rbenv dependencies for Debian $::osfamily.
 #
 class rbenv::deps::debian {
+  $libgdbm_package_name = $::os['distro']['codename'] ? {
```

Comment:
I believe you'll need to use the `$:: lsbdistcodename ` fact for this to pass specs https://puppet.com/docs/facter/3.1/core_facts.html#lsbdistcodename